<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SyntroHAServerClass</class>
 <widget class="QMainWindow" name="SyntroHAServerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SyntroHAServer</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAction">
    <property name="title">
     <string>Action</string>
    </property>
    <addaction name="actionBasicSetup"/>
    <addaction name="actionSerialPort"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuDevices">
    <property name="title">
     <string>Devices</string>
    </property>
    <addaction name="actionAddDevice"/>
    <addaction name="actionEditDevice"/>
    <addaction name="actionRemoveDevice"/>
   </widget>
   <widget class="QMenu" name="menuTimers">
    <property name="title">
     <string>Timers</string>
    </property>
    <addaction name="actionAddTimer"/>
    <addaction name="actionEditTimer"/>
    <addaction name="actionRemoveTimer"/>
   </widget>
   <addaction name="menuAction"/>
   <addaction name="menuDevices"/>
   <addaction name="menuTimers"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="mainToolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionParameters">
   <property name="text">
    <string>Parameters ...</string>
   </property>
  </action>
  <action name="actionBasicSetup">
   <property name="text">
    <string>Basic Setup</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionAddDevice">
   <property name="text">
    <string>Add Device</string>
   </property>
  </action>
  <action name="actionRemoveDevice">
   <property name="text">
    <string>Remove Device</string>
   </property>
  </action>
  <action name="actionAddTimer">
   <property name="text">
    <string>Add Timer</string>
   </property>
  </action>
  <action name="actionRemoveTimer">
   <property name="text">
    <string>Remove Timer</string>
   </property>
  </action>
  <action name="actionEditDevice">
   <property name="text">
    <string>Edit Device</string>
   </property>
  </action>
  <action name="actionEditTimer">
   <property name="text">
    <string>Edit Timer</string>
   </property>
  </action>
  <action name="actionSerialPort">
   <property name="text">
    <string>Serial port</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
